### PR TITLE
Kubeconfig configured

### LINKED ISSUE #2: Ask user if .kubeconfig is edited
Ask user if the correct .kubeconfig file is in the repository

### PR DESCRIPTION
**Pull request by wsf11**
Will ask the user if the config file is configured for kubernetes. Have to answer yes before preceding. Close #2

### CODE REVIEW

File: generators/app/prompt.js [hunk 1]
```diff
@@ -482,5 +495,6 @@ module.exports = {
    servicePrincipalId: servicePrincipalId,
    servicePrincipalKey: servicePrincipalKey,
    dockerRegistryPassword: dockerRegistryPassword,
-   dockerRegistryUsername: dockerRegistryUsername
+   dockerRegistryUsername: dockerRegistryUsername,
+   configUpdate:configUpdate
```
Comment (wesyao) [inline generators/app/prompt.js:499]: forgot a space after the colon

File: generators/app/utility.js [hunk 2]
```diff
@@ -283,6 +283,11 @@ function validateServicePrincipalKey(input) {
    return validateRequired(input, `You must provide a Service Principal Key`);
 }
 
+function validateConfigUpdate(input){
+   let valid = input.toLowerCase() === 'yes' ? input : '';
+   return validateRequired(valid, 'This file must be updated with the correct config file before you select "Yes" ');
```
Comment (wesyao) [inline generators/app/utility.js:288]: is there a unified place where all of these strings go?

**Review by wesyao — APPROVED** [review]

File: generators/app/utility.js (hunk 2 above)
Comment (wsf11) [reply generators/app/utility.js:288]: This is the format for all the other "validate...". It looks like all of the strings are just passed in like this. They are all just used as return messages for the validateRequired method

Comment (wesyao) [reply generators/app/utility.js:288]: Yup, then it makes sense to follow their pattern

**Pull request merged**